packages/chains/src/chapel/params.ts

```diff
@@ -5,7 +5,7 @@ import chainAddresses from "./addresses";
 
 const specificParams: ChainParams = {
   blocksPerYear: BigNumber.from((20 * 24 * 365 * 60).toString()),
-  cgId: "binancecoin",
+  cgId: "binancecoin-chapel",
```

Comment:
Had to change this, otherwise it clashes with the cgId of the actual BNB chain. It needs to be unqiue so it's not overriding BNB here:

https://github.com/Midas-Protocol/monorepo/blob/318683f1b3297ff7d87e99b1e3457ba616ff7b90/packages/ui/hooks/useUSDPrices.ts#L5-L11

Reply:
lovely solution :)